### ISSUE TITLE
Change default bokeh tools for histogram to be xpan, xwheel_zoom

### ISSUE THREAD

**Issue by timothydmorton** [ioam/holoviews#1777]
I find that it's much more useful to be able to pan/zoom in only the x-direction in a histogram.  I suggest that the default tools to appear with a histogram plot with bokeh should be the same that show up when I use `%%opts Histogram [tools=['xpan', 'xwheel_zoom']]`.

**Comment by jlstevens** [comment]
Setting plot options per element type is trivial and I do understand the rationale behind this suggestion so it is entirely possible to do.

My worry is that maybe it is best to have a consistent set of tools for all elements? Are there other elements that might have different appropriate defaults? A related worry is that the set of tools might change when building an overlay...

**Comment by timothydmorton** [comment]
I noticed in going through the [bounds example](http://holoviews.org/reference/streams/bokeh/Bounds.html) that something in there caused the selection box tool to appear in the bokeh tool list, where it's not usually a default in the other hv/bokeh plots I've been checking out.  That's what made me think that different elements might have different default tools.  

Regarding whether this would change for an overlay, it seems to me more of a special case for a histogram to be overlaid with something that would really want 2-d pan/zooming, in which case the tools could be easily changed by the user.  The 1-d action seems like a nicer default.

**Comment by jlstevens** [comment]
> Regarding whether this would change for an overlay, it seems to me more of a special case for a histogram to be overlaid with something that would really want 2-d pan/zooming, in which case the tools could be easily changed by the user. The 1-d action seems like a nicer default.

I agree for this specific case. I'm wondering about other elements that might potentially want a different set of default tools that can be overlaid.
